### PR TITLE
feat: added a new profile data

### PR DESCRIPTION
**Pull request by Aditya-150**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1726"></a>

### CODE REVIEW

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by vedanshi555 — APPROVED** [review]

File: public/data/Aditya-150.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Aditya Prasad Mohanty",
+    "type": "personal",
+    "bio": "Open Source Enthusiast | 🕷🕸🕷",
+    "avatar": "https://github.com/Aditya-150.png",
+    "links": [
+        {
+            "name": "Follow me on Github",
```
Comment (Panquesito7) [inline public/data/Aditya-150.json:8]: I recommend capitalizing the `h` here. What do you think?
```suggestion
            "name": "Follow me on GitHub",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Comment (Aditya-150) [reply public/data/Aditya-150.json:8]: > I recommend capitalizing the `h` here. What do you think?
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** ❓ I have a question

Ya, sure . I should have done that. Thanks for your recommendation.

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**